sdk/php/runtime/template/README.md

```diff
@@ -311,3 +311,48 @@ class Example
      // ...
 }
 ```
+
+#### Directories and Files
+
+Directories and Files have some additional metadata that can be specified
+
+##### Defaults
+
+Specifying a default value for a `Directory` or `File` isn't easily possible. 
+Instead you can specify a `DefaultPath` attribute with an absolute or relative path.
+If relative then the path is relative to the `source` specified in the `dagger.json` file.
+
+This tells Dagger; if no argument is specified, fetch the `Directory` or `File` from the `DefaultPath`.
+
+The following example shows how you would get the parent directory of your `source`:
+
+```php
+#[DaggerFunction]
+public function myDaggerFunction(
+    #[DefaultPath('../')]
+    Directory $dir,
+): Container {
+    // ...
+}
+```
+
+##### Ignore
+
+For Directories only, an `Ignore` Attribute to specify what files|directories should be ignored.
+This is done by providing an array of strings.
+Each string follows the [`.gitignore` syntax](https://git-scm.com/docs/gitignore/en): 
+prefixing a path with `!` will actually include it instead of ignoring it.
+
+The following example would ignore your `vendor/` and `tests/` directories:
+
+```php
+#[DaggerFunction]
+public function myDaggerFunction(
+    #[DefaultPath('../')]
```

Comment:
Should be `.` with https://github.com/dagger/dagger/pull/8391

Reply:
I've aimed at the parent directory again, maybe I should have one example as current dir and the other as parent dir... avoid lengthening the docs but still demonstrate both?